ledger/acctupdates.go

```diff
@@ -260,6 +255,8 @@ func (r resourcesUpdates) getForAddress(addr basics.Address) map[basics.Creatabl
 func (au *accountUpdates) initialize(cfg config.Local) {
```

Comment:
nit: same as before
```suggestion
func (au *accountUpdates) initialize(MaxAcctLookback, EnableAccountUpdatesStats, AccountUpdatesStatsInterval) {
```

Reply:
Same reasoning for the other tracker: all trackers follow this pattern.